dnf-behave-tests/features/search-sort.feature

```diff
@@ -0,0 +1,92 @@
+Feature: Sort search command output
+
+Background:
+  Given I use repository "dnf-ci-search"
```

Comment:
Could you name the repository according to the name of the feature file, i.e. `search-sort`? The `dnf-ci-` prefix is legacy for repositories that could conflict with the ones on the host system (although those shouldn't be there for the tests). For custom repositories for a given feature file, the convention is to name them after the feature file name.